lib/manager/terragrunt/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as hashicorpVersioning from '../../versioning/hashicorp';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  commitMessageTopic:
+    'Terragrunt {{managerData.terragruntDependencyType}} {{depNameShort}}',
```

Comment:
Won't this result in the commit messages and PR titles being a bit awkward? e.g. "Update Terragrunt gitTags foo/bar". Would "Terragrunt module" or "Terragrunt dependency" work better?

Reply:
Agree, updated to reflect that. See this PR: https://github.com/bgdanix/renovate-tst/pull/7